lab_5_scrapper/scrapper.py

```diff
@@ -127,11 +240,24 @@ def _extract_url(self, article_bs: BeautifulSoup) -> str:
         Returns:
             str: Url from HTML
         """
+        url = ''
+        link_tags = article_bs.find_all('a')
+        for tag in link_tags:
+            url = tag.get('href')
+        return url
 
     def find_articles(self) -> None:
         """
         Find articles.
         """
+        for seed_url in self.get_search_urls():
+            response = make_request(seed_url, self.config)
+            if response.status_code == 200:
+                html_content = response.text
+                article_bs = BeautifulSoup(html_content, 'html.parser')
```

Comment:
nice! have you used these libraries before? lxml is the recommended parser for quick processing

Reply:
Ok, I put the lxml parser. No, I have never used them